internal/service/fm.go

```diff
@@ -0,0 +1,58 @@
+package service
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+// saveToFile сохраняем данные в хранилище json
+func (us *URLShortener) saveToFile() {
```

Comment:
```suggestion
func (us *URLShortener) saveToFile() error {
```
Любые ошибки в go принято ловить и отправлять наверх. Наверху ее можно проигнорировать или залогировать или обработать каким-то еще способом, вплоть до падения приложения. Но общий подход остается неизменным. Только ошибки, идущие по статусу warning, можно игнорить на месте, но всегда надо отдавать себе отчет, почему мы это делаем.